docs/assets/js/main.js

```diff
@@ -29,29 +30,14 @@ if ( tabs && tabs.length > 0 ) {
   }
 }
 
-const HIDDEN_CLASS = 'u-hidden';
-
 /**
  * @param {MouseEvent} event - The mouse event object from the click.
  */
 function handleDocumentClick( event ) {
   const target = event.target;
-  if ( !target.matches( '[data-toggle-code]' ) ) {
-    return;
-  }
-  event.preventDefault();
-  const container = target.parentNode;
-  const codeEl = document.querySelector( target.getAttribute( 'href' ) );
-  if ( codeEl && codeEl.classList.contains( HIDDEN_CLASS ) ) {
-    codeEl.classList.remove( HIDDEN_CLASS );
-    container.querySelector( '[data-toggle-code="hide"]' ).classList.remove( HIDDEN_CLASS );
-    container.querySelector( '[data-toggle-code="show"]' ).classList.add( HIDDEN_CLASS );
-  } else {
-    codeEl.classList.add( HIDDEN_CLASS );
-    container.querySelector( '[data-toggle-code="hide"]' ).classList.add( HIDDEN_CLASS );
-    container.querySelector( '[data-toggle-code="show"]' ).classList.remove( HIDDEN_CLASS );
+  if ( target.matches( '[data-toggle-code]' ) ) {
```

Comment:
Would it make sense to confine all references to `[data-toggle-code]` to the same module? Right now there's a reference to it in `main.js`, `toggle-details.js`, and `variationPreviewTemplate.js`. Maybe it's looked up in toggle-details, which stores DOM references that the other modules then reference.

Reply:
I like this line of thought. It's tricky because we use the functionality in the CMS and in the static site which are very different environments. I think for now I'll just export the attribute selector `data-toggle-code` so that it's only defined in one place and the separate files can import it.